ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiParticipantPruningService.scala

```diff
@@ -70,12 +73,20 @@ final class ApiParticipantPruningService private (
 
               // If write service pruning succeeds but ledger api server index pruning fails, the user can bring the
               // systems back in sync by reissuing the prune request at the currently specified or later offset.
-              _ <- pruneWriteService(pruneUpTo, submissionId, request.pruneAllDivulgedContracts)
-
-              pruneResponse <- pruneLedgerApiServerIndex(
-                pruneUpTo,
-                request.pruneAllDivulgedContracts,
-              )
+              _ <- Tracked.future(
+                metrics.daml.services.pruning.pruneStarted,
+                metrics.daml.services.pruning.pruneCompleted,
+                pruneWriteService(pruneUpTo, submissionId, request.pruneAllDivulgedContracts),
+              )(MetricsContext(("phase", "writeService")))
```

Comment:
To make the labels easier to understand for operators, I'd suggest calling this phase `underlyingLedger` or simply `ledger` instead (i.e., `writeService` is an internal interface name).
@hrischuk-da may have additional insight on this.